packages/labnative/pages/Card.jsx

```diff
@@ -107,7 +107,11 @@ const CardPage = () => {
           >
             <PlanCard.Subtitle>Get access to</PlanCard.Subtitle>
             <PlanCard.List>
-              <PlanCard.ListItem icon={MapPin} text="2.900 gyms and studios" />
+              <PlanCard.ListItem
+                color="#6B6B78"
```

Comment:
It's better to follow the preference order below, from left to right:

| theme color | token | hex |
| ------------ | ------ | --- |
| `text.secondary` | `deep` | `#6B6B78 ` |